packages/stores/src/ui-config/manage-liquidity/add-concentrated-liquidity.ts

```diff
@@ -271,11 +272,13 @@ export class ObservableAddConcentratedLiquidityConfig extends TxChainSetter {
   };
 
   get range(): [Dec, Dec] {
```

Comment:
This can be decorated with `@computed`

Tip: my general rule is anything that isn't a direct access to a observable, or involves any computation at all, should be a `@computed` property